packages/IMAPClient-Core.package/ICFileReadWriter.class/instance/readEmailsFromDisk.st

```diff
@@ -0,0 +1,6 @@
+read file
+readEmailsFromDisk
+
+	self rootDirectory
+			assureExistence; 
```

Comment:
Test for that would be nice.

Reply:
I don't get, what you mean. This is taken from the standard library.